src/components/Project/test/ProjectDashboard.test.tsx

```diff
@@ -69,6 +70,8 @@ describe('ProjectDashboard', () => {
     [sortQueryKeys.key]: executionSortFields.createdAt,
   };
 
+  jest.spyOn(LocalCache, 'useLocalCache');
```

Comment:
Is this needed? don't see where the function return or isCalled checked or mocked.
Looks like without it tests should also pass

Reply:
`useLocalCache` was added to a child component `DomainSettingsSection` and tests fail when parent tries to render a child